zcl_text2tab_parser.clas.abap

```diff
@@ -239,6 +255,18 @@ method CREATE.
     ro_parser->mv_date_format = i_date_format.
   endif.
 
+  CLEAR ro_parser->mv_begin_comment.
+  IF i_begin_comment IS SUPPLIED.
+    ro_parser->mv_begin_comment = i_begin_comment.
+  ELSE.
+    SELECT low UP TO 1 ROWS FROM tvarvc INTO ro_parser->mv_begin_comment
+      WHERE name = 'ZMOCKUP_LOADER_COMMENT' AND type = 'P'.
+    ENDSELECT.
```

Comment:
let's do it in another PR. If tvarvc feature is implemented it should be for all params, not just for this one. Also when you do, could you pls use `select single`. And one question to consider - unit tests for tvarvc ... the selects should not affect unit tests. Let's discuss in a separate PR or issue pls.